src/test/java/io/cdap/plugin/successfactors/source/metadata/SuccessFactorsSchemaGeneratorTest.java

```diff
@@ -74,8 +78,12 @@ public void testSelectWithExpandNames() throws SuccessFactorsServiceException {
 
   @Test
   public void testBuildExpandOutputSchema() throws SuccessFactorsServiceException {
+    SuccessFactorsPluginConfig pluginConfig = new SuccessFactorsPluginConfig("referenceName",
+      "baseUR", "entityName", "associateEntityName", "username",
+      "password", "filterOption", "selectOption", "expandOption",
+      "paginationType");
     Schema outputSchema = generator.buildExpandOutputSchema("Benefit",
-                                                            "eligibleBenefits");
+                                                            "eligibleBenefits", "assEntity", pluginConfig);
```

Comment:
Can we use a better test string for associatedEntity? 😅
A suggestion is to keep the string as "associatedEntity" itself

Reply:
Sure, done.